docs/features.rst

```diff
@@ -7,8 +7,100 @@ Writing Features
 The standard `Gherkin syntax`_ is supported, including scenario
 outlines, doc strings, data tables and internationalization.
 
-Feature and scenario tags can be used and are converted to Nose tags, see
-docs for the `Attribute selector plugin`_.
+Feature
+-------
+
+A feature is a single file that typically defines a single story.
+It has a name and an optional description, an optional background_ and many
+scenarios.
+
+.. code-block:: gherkin
+
+    Feature: Search
+
+        As a user
+        I want to do a search for something not in the default categories
+        So that I can provide more detailed search parameters
+
+A feature may also have tags_.
+
+Background
+----------
+
+The background_ is an optional section that is run before every scenario and
+contains steps. It is used to set up fixtures common to each scenario_ of
+the feature_.
+
+A background does not have a name or tags.
+
+If a step fails during the background all the scenarios of this feature_ will
+fail.
```

Comment:
This is wrong. A (non-deterministic, which is bad) background can fail for one scenario and succeed for another.